src/handlers/chain_agnostic/assets.rs

```diff
@@ -55,7 +59,11 @@ pub static BRIDGING_ASSETS: phf::Map<&'static str, AssetEntry> = phf_map! {
             decimals: 6,
         },
         simulation: SimulationParams {
-            balance_storage_slot: 0,
+            // Must be in sync with the `USDT_CONTRACTS` from above
+            balance_storage_slots: &phf_map! {
+                "eip155:10" => 0u64,
```

Comment:
Are we missing Base here?

```suggestion
                "eip155:8453" => 0u64,
```

Reply:
unfortunately, Bungee supports only USDC on Base